packages/transitive-overlay/src/TransitiveOverlay.story.js

```diff
@@ -26,6 +31,11 @@ const walkTransitWalkItinerary = require("@opentripplanner/itinerary-body/src/__
 const walkTransitWalkItineraryNoIntermediateStops = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-transit-walk-no-intermediate-stops.json");
 const walkTransitWalkTransitWalkItinerary = require("@opentripplanner/itinerary-body/src/__mocks__/itineraries/walk-transit-walk-transit-walk.json");
 
+// Add routeShortName to walk-transit-walk itineary here
+// to illustrate labeledModes in the transitive overlay
+// (and to not interfere with other stories that use that itinerary).
+walkTransitWalkItinerary.legs[1].routeShortName = "Blue";
```

Comment:
Off the top of my head, I worry about how controlling the label content via routeShortName will work in the real world, Bihn.

 - hoping there's another way to change the content of the route label content in OTP-UI + transitive-js beyond monkey-patching routeShortName 
 - was imagining my client would provide OTP-UI an optional slot or callback, which transitive-js call whenever  rendering a route label:
   function renderRouteLabelSlot(itineraryLeg) {
       if (itineraryLeg.mode === "TRAM") return "MAX";
       if (itineraryLeg.mode === "RAIL") return "WES";
       if (itineraryLeg.mode === "BUS") return itineraryLeg.routeShortName; 
       return null;  // null or undefined or empty string will tell transitive-js not to render a route label
  }

Reply:
@fpurcell and all, please see #248 as an option to accommodate the custom route label function you suggested above. I guess there is no need to change anything in transitive.js at this time with this approach.